composer.json

```diff
@@ -137,18 +136,18 @@
     "twig/twig": ">=3.9.0 <3.14.0"
   },
   "require-dev": {
+    "chrome-php/chrome": "^1.4.0",
     "codeception/codeception": "^5.0.3",
     "codeception/module-symfony": "^3.1.0",
+    "composer/composer": "*",
     "ergebnis/phpstan-rules": "^2.0",
-    "phpstan/phpstan": "1.12.5",
-    "phpstan/phpstan-symfony": "^1.3.5",
-    "phpunit/phpunit": "^9.3",
     "gotenberg/gotenberg-php": "^1.1 || ^2.4",
-    "composer/composer": "*",
-    "chrome-php/chrome": "^1.4.0",
-    "webmozarts/console-parallelization": "^2.1",
+    "phpstan/phpstan": "*",
```

Comment:
@michalananapps why do you also change the phpstan version? It is fixed for a reason :)

Reply:
Sorry. I didn't properly clear the previous changes after previous experiments :).

Fixed